gst/nnstreamer/meson.build

```diff
@@ -45,6 +45,7 @@ endforeach
 nnst_plugins = [
   'tensor_aggregator',
   'tensor_converter',
+  'tensor_crop',
```

Comment:
Any reason not using `tensor_transform`? Are you planning sub-plugins of `tensor-crop`?